scripts/check_lint.py

```diff
@@ -195,12 +195,19 @@ def make_path():
 
   # In addition, add any bin directories near the lib directories in the python
   # path. This makes it possible to find flake8 in ~/Library/Python/2.7/bin
-  # after pip install --user flake8.
-  lib_pattern = re.compile(r'(.*)/lib/')
+  # after pip install --user flake8. Also handle installations on Windows which
+  # go in %APPDATA%/Python/Scripts.
+  lib_pattern = re.compile(r'(.*)/[^/]*/site-packages')
   for entry in sys.path:
+    entry = entry.replace(os.sep, '/')
```

Comment:
Ick. You *could* just force everyone to add the directory containing the flake8 binary to their $PATH. But I suppose what you've got is slightly better (perhaps much better on Windows.)

Reply:
I'd prefer to not require setting `PATH` so that `pip install --user flake8` just works.